clang-tools-extra/build/CMakeFiles/3.22.1/CMakeCCompiler.cmake

```diff
@@ -0,0 +1,72 @@
+set(CMAKE_C_COMPILER "/usr/bin/cc")
```

Comment:
Why does this PR touch any CMake files? Is it necessary for the intended change of behavior of one or two AST matchers?

Reply:
Also, why are we adding binaries?

Reply:
Looks like simple mistake, plee remove that build directory from a change.

Reply:
Yes, removing it.